BookkeeperAPI/Program.cs

```diff
@@ -1,18 +1,33 @@
-using Microsoft.Extensions.Options;
+using BookkeeperAPI.Data;
+using Microsoft.EntityFrameworkCore;
 using Microsoft.OpenApi.Models;
 using System.Reflection;
+using Newtonsoft.Json;
 
 var builder = WebApplication.CreateBuilder(args);
+IConfiguration configuration = builder.Configuration;
 
 // Add services to the container.
 
-builder.Services.AddControllers();
+builder.Services.AddControllers()
+    .AddNewtonsoftJson(options =>
+    {
+        options.SerializerSettings.ReferenceLoopHandling = ReferenceLoopHandling.Ignore;
+    }); ;
+
+// Add DB context to connect to database
+builder.Services.AddDbContext<BookkeeperContext>(
+    optionsBuilder => optionsBuilder
+    .UseNpgsql(configuration["ConnectionStrings:BookkeeperDB"])
```

Comment:
value for `ConnectionStrings:BookkeeperDB` is missing from appsettings.json. We should store in a keyvault(If possible).

Reply:
currently, this is being stored in appsettings.Development.json, will create keyvault soon and integrate it in current build

Reply:
To track keyvault progress, created a task, [BOOKA-39](https://bookkeeper-manager.atlassian.net/browse/BOOKA-39)